scripts/installation/bin/yb-ctl

```diff
@@ -1060,6 +1065,13 @@ class ClusterControl:
 
             subparsers[cmd_name] = self.setup_base_parser(cmd_name, help=help)
 
+        # Commands that take -f or --force flag
+        force_commands = ["destroy", "stop", "stop_node", "restart"]
+        for cmd in force_commands:
+            subparsers[cmd].add_argument("-f","--force",
+                                        action='store_true', 
+                                        help="%s forcefully" % cmd,)
```

Comment:
```suggestion
                                         action='store_true', 
                                         help="%s forcefully" % cmd,)
```

Please indent so that the args line up to `(`.